docs/policy-provider.md

```diff
@@ -0,0 +1,50 @@
+# Policy Providers
+
+Ratify implements an extensible policy provider interface allowing for different policy providers to be created and registered. The policy provider to be used is determined by the policy plugin specified in the `policies` section of the configuration. 
+
+Currently, Ratify supports a Configuration based Policy Provider named `configPolicy`.
+
+## Sample Config
+
+```
+...
+"policies": {
+    "version": "1.0.0",
+    "plugin": {
+        "name": "configPolicy",
+        "artifactVerificationPolicies": {
+            "application/vnd.cncf.notary.v2.signature": "all"
+        }
+    }
+},
+...
+```
+
+- The `name` field is REQUIRED and MUST match the name of the registered policy provider
+- `artifactVerificationPolicies`: map of artifact type to policy; each entry in the map's policy must be satisfied for Ratify to return true.
+    - `any`: policy that REQUIRES at least one artifact of specified type to verify to `true` 
+    - `all`: policy that REQUIRES all artifacts of specified type to verify to `true`
+    - `none`: policy that does NOT require any artifacts to verify to `true`
+- Default policy:
+    - The `default` policy applies to unspecified artifact types. The `default` policy is set to `none`. Thus, all unspecified artifact types will be ignored in determing overall success result.
+    - The `default` policy can be overriden to `any` or `all` in the map:
```

Comment:
Suggestion to provide some examples for any. If 'any' is configured, does it required 1+ of a specific type to be valid to produce a success result? or does the artifact type not matter

Reply:
Added some examples demonstrating. Could you take another look at them @susanshi. Not sure if they are confusing.